src/TensorFlowNET.Core/Status/Status.cs

```diff
@@ -25,26 +26,37 @@ namespace Tensorflow
     /// TF_Status holds error information. It either has an OK code, or
     /// else an error code with an associated error message.
     /// </summary>
-    public class Status : DisposableObject
```

Comment:
📝 `DisposableObject` follows legacy resource management patterns that are no longer recommended for use. As types are migrated to new patterns, they will no longer use `DisposableObject`, and eventually the type can simply be deleted.